[
    {
        "content": "<p>There's a question on the CESM bulletin board: <a href=\"https://bb.cgd.ucar.edu/cesm/threads/help-with-python-polar-stereographic-plotting-of-cice-and-pop-data.7319/\">https://bb.cgd.ucar.edu/cesm/threads/help-with-python-polar-stereographic-plotting-of-cice-and-pop-data.7319/</a> -- basically, the user has copied CESM output from cheyenne to a local machine. Reading in 30 years of model output and plotting fields from CAM and CICE is fine but the following error occurs when trying to plot a POP field: <a href=\"https://bb.cgd.ucar.edu/cesm/attachments/screenshot-from-2022-05-05-11-33-17-png.2363/\">link to her screenshot of error</a></p>\n<p>I've invited the user to join Zulip, so I don't know if the conversation will eventually move here or to <a class=\"stream\" data-stream-id=\"10\" href=\"/#narrow/stream/10-python-questions\">#python-questions</a> , but if any of you can help with the forum post that would be great!</p>",
        "id": 54586,
        "sender_full_name": "Michael Levy",
        "timestamp": 1651766908
    }
]